### PR TITLE
AV_GO

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Makes Miners Mine Soup and bring it back to HQ

File: src/BotGO/RobotPlayer.java [hunk 1]
```diff
@@ -69,20 +68,46 @@ static void runHQ() throws GameActionException {
...
+
         tryBlockchain();
-        tryMove(randomDirection());
-        if (tryMove(randomDirection()))
-            System.out.println("I moved!");
         // tryBuild(randomSpawnedByMiner(), randomDirection());
-        for (Direction dir : directions)
-            tryBuild(RobotType.FULFILLMENT_CENTER, dir);
+        //for (Direction dir : directions)
+        //    tryBuild(RobotType.FULFILLMENT_CENTER, dir);
```
Comment (KyleMcB) [inline src/BotGO/RobotPlayer.java:90]: leaving inactive code in comments can be confusing to people reading code.

**Review by KyleMcB — APPROVED** [review]
Thanks for making the change I asked for in the other pull request.

File: src/BotGO/RobotPlayer.java (hunk 1 above)
Comment (ghost) [reply src/BotGO/RobotPlayer.java:90]: I kept it in comments since I thought I might use it again.

**Comment by KyleMcB** [comment]
Starting over. Must use Trello

**Pull request closed without merging**